### DISCUSSION TITLE
Have co-authored commits be the default setting for users

### DISCUSSION THREAD

**Discussion by BrookJeynes** [tinacms/tinacms#4447] [Ideas]
CC: @JackDevAU @adamcogan @wicksipedia @bradystroud

Commits aren't co-authored by default. This option needs to be switched on for every user.

Suggestion: Make this opt-out instead of opt-in.

**Comment by kldavis4** [comment]
I think the issue with this is that we only have the email that the user registered with in Tina Cloud and we can't assume that the user wants their personal or work email associated with their git commits - see the [FAQ](kldavis4/feat/ENG-1202/separate_content_repo_support). 

Note that if the user registers to tina cloud via github login this shouldn't be a problem)  I think, ideally, you'd want to have a step during registration where the user is asked if they want this turned on and then they are asked to sign in to github and then tina cloud can determine their anonymous github email.
